### PR TITLE
site-admin: Show username if displayName is null

### PR DESCRIPTION
**Pull request by indradhanush**
Fixes a bug when nothing is seen if the user does not have a display name. Example can be seen on S2:

image

**Storybook**

CleanShot 2023-04-05 at 17 09 05

## Test plan

1. Tested locally
1. Builds should pass

## App preview:

- [Web](https://sg-web-ig-bug-site-config-username.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

**Review by cbart — APPROVED** [review]
While this is a very simple change, and I think we might land it as is...

...would you like to add a _unit test_?

**Comment by mrnugget** [comment]
> ...would you like to add a _unit test_?

I think the better approach is to add a storybook here. Allows us to quickly test this with different scenarios (like a unit test) and we can look at it! cc @sashaostrikov our storybook expert here

**Review by sashaostrikov — APPROVED** [review]
LGTM, but let's add some storybook coverage for your component in this commit!

**Comment by indradhanush** [comment]
Added a story book as suggested.

**Review by sashaostrikov — APPROVED** [review]
LGTM! Added a comment about a story name

File: client/web/src/site-admin/SiteConfigurationHistoryItem.story.tsx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+import { SiteConfigurationHistoryItem } from './SiteConfigurationChangeList'
+
+const config: Meta = {
+    title: 'web/site-admin/SiteConfigurationHistoryItem',
+    component: SiteConfigurationHistoryItem,
+}
+
+export default config
+
+export const HistoryItemWithNoAuthor: Story = () => {
```
[the remark this answers is not in the document]
Comment (indradhanush) [reply client/web/src/site-admin/SiteConfigurationHistoryItem.story.tsx:14]: I wondered about that, but then saw the story name was generated from the variable itself. Is that a suggested convention that should be followed even if the name I want to add is "History item with no author" in this case? 

image

[the remark this answers is not in the document]
Comment (sashaostrikov) [reply client/web/src/site-admin/SiteConfigurationHistoryItem.story.tsx:14]: Yeah, that's true, actually I don't know about a convention, but I just add it all the time myself :) if it is done everywhere, we can keep it for the sake of uniformity

**Pull request merged**